modules/system/database/migrations/2022_08_06_000026_Db_System_Add_App_Birthday_Date.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Winter\Storm\Database\Updates\Migration;
+use Carbon\Carbon;
+use System\Models\Parameter;
+use System\Models\PluginVersion;
+
+class DbSystemAddAppBirthdayDate extends Migration
```

Comment:
We can use anonymous classes for migrations now.
```suggestion
return new class extends extends Migration
```